docs/connect-pmm.md

```diff
@@ -17,23 +21,36 @@ To ensure that Percona Platform can establish a connection with the PMM instance
 3. Save the changes.
 
 ### Check that you are a member of an existing organization
-1. Log in to [Percona Platform](https://portal.percona.com) using your Percona Account.
+1. Log in to [Percona Platform](https://portal.percona.com). 
+   
+  !!! hint alert alert-success "Tip"
+    If you are connecting via GitHub, make sure that you set your email address as **public** in your GitHub account. If your email address is private instead, Percona Platform cannot access it to authenticate you.
```

Comment:
```suggestion
    If you are connecting via GitHub, make sure that you set your email address as **public** in your GitHub account. Otherwise, Percona Platform cannot access it to authenticate you.
```

Reply:
It isn't only about being private, there are a few other options that will also prevent the email address to be public, thus the "Otherwise"